src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt

```diff
@@ -36,10 +37,24 @@ class VarselService(
         behandlerRef: String?,
         behandlerDocument: List<DocumentComponentDTO>,
         behandlerPdf: ByteArray?,
-        behandlerbrevId: Pair<Int, UUID>?,
+        behandlerbrevId: UUID?,
         behandlerbrevParentId: String?,
         behandlerInnkallingUuid: UUID?,
     ) {
+        if (narmesteLeder != null) {
+            narmesteLederVarselService.sendVarsel(
+                createdAt = tidspunktForVarsel,
+                moteTidspunkt = moteTidspunkt,
+                narmesteLeder = narmesteLeder,
+                varseltype = varselType,
+            )
+        } else {
+            val altinnMelding = createAltinnMelding(virksomhetsbrevId, virksomhetsnummer, virksomhetsPdf, varselType)
```

Comment:
Det er ikke slik at vi vil ha i pose og sekk og sende altinn-melding uansett? Eller blir det for mye mas?

Reply:
Ja, det kan kanskje bli for mye mas. Så vi begynner med kun de uten nærmeste leder, og så kanskje referat til alle i neste omgang 👍

Reply:
Lisa har en mening og en plan på dette for å også ta vare på arbeidsgivere, så vi er ikke helt ferdig med å diskutere.